interfaces/builtin/opengl.go

```diff
@@ -74,6 +74,12 @@ unix (send, receive) type=dgram peer=(addr="@nvidia[0-9a-f]*"),
 @{PROC}/devices r,
 /sys/devices/system/memory/block_size_bytes r,
 unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
+/dev/nvidia-uvm rw,
+/dev/nvidia-uvm-tools rw,
+/dev/nvidiactl rw,
+/dev/nvidia* rw,
```

Comment:
Already included here: https://github.com/snapcore/snapd/pull/5696/commits/9c1eaf07eae5923c795032526300f0ddf1e55e72#diff-6b578415ded16b60039c76b7f0ef8eacR66

Reply:
I agree with Maciej

Reply:
I removed the duplicated rules